### DISCUSSION TITLE
[Feature] Option to display filename in gallery

### DISCUSSION THREAD

**Discussion by NHAS** [immich-app/immich#3980] [Feature Request]
### The feature

Hi there (Thanks for making immich is fantastic!), 

I'd quite like to be able to see said filenames in the photo gallery so I can see what I've named the media files (especially useful after importing a large number of already named/numbered media files). 

This feature request can also be thought of "can we have options to show metadata in the photo gallery"

Thank you!

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
If you click the i icon in the top right it will open the metadata panel.

**Reply by NHAS** [reply]
Hi there, Im aware of that. Im meaning in the gallery view itself while you're scrolling through photos.

**Reply by bo0tzz** [reply]
How do you envision that looking?

**Reply by NHAS** [reply]
Bearing in mind Im not a web developer:

![image](https://github.com/immich-app/immich/assets/6820641/21ca3e60-8e2f-4787-a896-f2d909f7a26e)

**Reply by NHAS** [reply]
Its a little bit crowded like this, but being able to see name of the file or some level of description might be useful!

**Reply by bo0tzz** [reply]
@alextran1502 thoughts on this FR?

**Reply by alextran1502** [reply]
I don't like the look of showing name of the file like that. I will have to reject the idea

**Reply by NHAS** [reply]
Aw okay, any other on-gallery display of the filename would also be neat! But I can understand if this isnt what you're looking to do. 

Thanks!
